README.md

```diff
@@ -30,26 +30,42 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+The biggest difference between .forEach and .map is that .map creates a new array automatically while with .forEach you have to manually establish a new array.
+
 2. What is the difference between a function and a method?
 
+functions are actions that intake data, do something, and then return a result. Methods take functions and applythem to a given object (object, array, class, etc.).
```

Comment:
Functions are just a Javascript object that perform a task (although the technically can be empty).  Methods are functions that belong in an object's property/prototype.  Methods can use the an object's properties but does not have to.